src/content/post/surfersol_blogpost.md

```diff
@@ -0,0 +1,34 @@
+---
+publishDate: 2023-11-01T00:00:00Z
+title: October's Meeting Minutes
+image: /images/NovemberNOSTR.webp
+description: October's Meeting Minutes
+category: Beginner
+author: SurferSol 
+tags:
+  - lightning-network
+  - Git workflow
+  - Blogpost
+---
+
+# October's Meeting Minutes
+
+Meeting’s announcements and discussions
+- [Nostr](https://testnet.plebnet.dev/nostrrelay/1) 
+| [Resources](https://metadata.plebnet.dev/)
+
+- P2P channel – To buy, sell, trade, and offer services in exchange for Bitcoin or Lightnting.  Listing your offers here for exchange in the members group.  Buyers and sellers contact each other directly via Private Message.  The marketplace is not to be used for blatant illegal activity.
+
+- Future Projects – GetAlby to announce a long term project with collaboration with Plebnet.dev
+
+- Recognizing our Active Contributors - Blogging Bitcoin, Saucy_tech, Tonyboi, Coinguybri, Fenix, Ash / Smash, BitKarrot
+
+- [Nostr use cases and benefits and Nostr matrix client list](https://nostorg.github.io/clients/)
+
+- Contributor’s projects outside of Plebnet.dev of personal development progress
+
+- [Completion of a LNBITS or GetAlby Pull Request would gets a free 2024 Bitcoin Tix Open Source](https://b.tc/conference/opensource)
```

Comment:
Not sure if getAlby is on the list. I think its only LNBits but please cross check. thanks